libs-haskell/test-utils/DA/Test/Sandbox.hs

```diff
@@ -167,6 +169,8 @@ getCantonConfig conf@SandboxConfig{..} portFile mCerts (ledgerPort, adminPort, d
                 , [ "clock" Aeson..= Aeson.object
                         [ "type" Aeson..= ("sim-clock" :: T.Text) ]
                   | Static <- [timeMode] ]
+                , [ "dev-version-support" Aeson..= devVersionSupport]
+                , [ "non-standard-config" Aeson..= True]
```

Comment:
does this one always need to be `true`? it feels a bit off

Reply:
Good catch! Done.